packages/api/src/service/alert/alert.service.ts

```diff
@@ -37,18 +36,18 @@ const direcctDomainName = "direccte.gouv.fr";
 
 const block = (alert: Alert) => {
   alert.email_state = "blocked";
-  alert.email_processed_at = new Date().getTime();
+  alert.email_processed_at = new Date();
 };
 
 class AlertService {
-  public deleteAll(): Observable<DeletedData[]> {
+  public deleteAll(): Observable<Number> {
```

Comment:
eslint/tslint prefer 
```suggestion
  public deleteAll(): Observable<number> {
```